### ISSUE TITLE
Age backend: ssh password requested twice

### ISSUE THREAD

**Issue by yml** [gopasspw/gopass#1553]
### Steps To Reproduce

* `./gopass init --storage fs --crypto age`
* select a private key for encrypting secrets  `age - ssh-rsa` (this key is password protected
* create a new account for titi.com `./gopass create` you need to provide the (ssh key paswword one time)
* display the password `./gopass show titi` 

The Issue in the last step is that you need to enter your ssh password 2 times

### Expected behavior

Enter the ssh password only **one time**

### Environment

- OS:Ubuntu 18.04.5 LTS
- gopass Version:
```
gopass 1.10.0+a6031515 (a6031515 2020-08-24 11:09:18) go1.15 linux amd64
<root>     -  age 0.0.1 -   fs 0.1.0
Available Crypto Backends: age, gpgcli, plain
Available Storage Backends: fs, gitfs, ondisk
```
- Installation method: source

**Comment by dominikschulz** [comment]
Yes, this is currently a limitation. Thanks for reporting this.

The reason for that is that both the age keyring as well as the secret are encrypted.

We'll eventually want to fix this, but this is rather low priority.
However please continue to test the new backend and report bugs.

**Comment by AnomalRoil** [comment]
This seems to have been fixed aready, probably with #2060 

Closing.

**Comment by dominikschulz** [comment]
Yes, this should definitely be fixed by now. Thanks for cleaning up this issue.
